backend/app/utilities/firebase_init.py

```diff
@@ -0,0 +1,11 @@
+import os
+
+import firebase_admin
+from firebase_admin import credentials
+
+
+def initialize_firebase():
+    current_dir = os.path.dirname(os.path.abspath(__file__))
+    service_account_path = os.path.join(current_dir, "serviceAccountKey.json")
```

Comment:
Can we keep all the credentials in one area? Let's keep this in the base backend directory.